src/daemon/ca/certauth.rs

```diff
@@ -1622,16 +1627,25 @@ impl CertAuth {
     }
 
     /// Renew existing ROA objects if needed.
-    pub fn route_authorizations_renew(&self, config: &Config, signer: &KrillSigner) -> KrillResult<Vec<CaEvt>> {
+    pub fn route_authorizations_renew(
+        &self,
+        force: bool,
+        config: &Config,
+        signer: &KrillSigner,
+    ) -> KrillResult<Vec<CaEvt>> {
         let mut evt_dets = vec![];
 
         for (rcn, rc) in self.resources.iter() {
-            let updates = rc.renew_roas(&config.issuance_timing, signer)?;
+            let updates = rc.renew_roas(force, &config.issuance_timing, signer)?;
             if updates.contains_changes() {
-                info!(
-                    "CA '{}' reissued ROAs under RC '{}' before they would expire: {}",
-                    self.handle, rcn, updates
-                );
+                if force {
+                    info!("CA '{}' reissued all ROAs under RC '{}'", self.handle, rcn);
```

Comment:
The strength of the term "force" makes me wonder if "info" is suitable here or should this be "warn" ?

Reply:
No "warn" would make users worry and this is by design